_inc/jetpack-modules.js

```diff
@@ -59,6 +59,15 @@
 		$( '.modal' ).focus();
 	};
 
+	/**
+	 * If modalinfo is defined, auto popup the modal
+	 */
+	$( window ).load(function() {
```

Comment:
Are you sure we need to use `$( window ).load` here instead of `$( document ).ready`?